src/aparna/nestedSwitchCaseForBank/NestedSwitchCaseForBank.java

```diff
@@ -0,0 +1,178 @@
+/*Assignment 15:-
+ * A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI
+ * 
+ */
+package aparna.nestedSwitchCaseForBank;
+
+public class NestedSwitchCaseForBank {
+
+	String customerName;
+	double totalBalanceICICI, totalBalanceHDFC, totalBalanceBOB, totalBalanceSBI;
+	int debitCountICICI, debitCountHDFC, debitCountBOB, debitCountSBI, creditCountICICI, creditCountHDFC,
+			creditCountBOB, creditCountSBI;
+
+	void setinitialBalanceNCustomerName(String name, double balanceICICI, double balanceHDFC, double balanceBOB,
+			double balanceSBI) {
+		customerName = name;
+		totalBalanceICICI = balanceICICI;
+		totalBalanceHDFC = balanceHDFC;
+		totalBalanceBOB = balanceBOB;
+		totalBalanceSBI = balanceSBI;
+		System.out.println("Customer " + customerName + " Bank Transaction Details are as follows: ");
+		System.out.println("Total Bank Balance in ICICI Bank: " + totalBalanceICICI
+				+ " , Total Bank Balance in HDFC Bank: " + totalBalanceHDFC + " , Total Bank Balance in BOB Bank: "
+				+ totalBalanceBOB + ", Total Bank Balance in SBI Bank: " + totalBalanceSBI);
+	}
+
+	void bankTransactionDetails(String operation, String bank, int amount) {
+		operation = operation.toUpperCase();
+		bank = bank.toUpperCase();
+
+		switch (operation) {
+		case "DEBIT":
+			switch (bank) {
+			case "ICICI":
+				if (totalBalanceICICI >= amount) {
```

Comment:
instead of writing same code block for multiple times, you can write debit() method which accept the two arguments and you can simply call that method.